java/src/main/bin/build.sh

```diff
@@ -147,6 +147,9 @@ install_brew_app "docker"
 docker-machine create --driver virtualbox default
```

Comment:
That's a good point, and I think worthy of commenting.  Honestly this file is only intended to be run on a Mac.  I would love a more general script, but since I can't figure out a way to build on OSX unless the native OS is OSX we're stuck with this setup.